rules-reviewed/jboss-eap/weblogic/weblogic-xml-descriptors.windup.xml

```diff
@@ -0,0 +1,282 @@
+                            This element in the weblogic-ejb-jar.xml file, which defaults to True, 
+                            is used for performance reasons to delay updates to the persistent store of all beans 
+                            until the end of the transaction. 
+                            When set to False, updates are sent to the database after each method invocation,
+                             but are not committed until the end of the transaction. 
+                             This allows other processes to access the persisted data while the transaction is waiting to be completed. 
+                             In JBoss EAP 6, you can achieve the same behavior by specifying the &amp;lt;sync-on-commit-only&amp;gt; in jbosscmp-jdbc.xml file.
+                        </message>
+                        <link href="https://access.redhat.com/articles/1326823" description="Map delay-updates-until-end-of-tx element from weblogic-ejb-jar.xml Elements to the JBoss Enterprise Application Platform Equivalent" />
+                        <tag>performance</tag>
+                        <tag>ejb</tag>
+                        <tag>database</tag>
+                    </hint>
+                </iteration>
+            </perform>
+        </rule>
+        <rule id="weblogic-xml-descriptor-04000">
+            <when>
+                <xmlfile as="default" matches="/weblogic-ejb-jar">
+                </xmlfile>
+            </when>
+            <perform>
+                <iteration>
+                    <classification title="WebLogic EJB XML" severity="mandatory" effort="3">
+                        <link href="https://access.redhat.com/articles/1326823" description="Map weblogic-ejb-jar.xml Elements to the jboss-ejb3.xml Descriptor" />
+                    </classification>
+                    <!-- TODO verify it should work in EAP6 -->
+                    <xslt description="JBoss EJB Descriptor (Windup-Generated)" template="../../../rules/jboss-eap/weblogic/transformations/xslt/weblogic-ejb-to-jboss.xsl" extension="-jboss.xml"/>
+                </iteration>
+            </perform>
+        </rule>
+        <rule id="weblogic-xml-descriptor-05000">
+            <when>
+                <xmlfile as="default" matches="/wl9:weblogic-ejb-jar">
+                    <namespace prefix="wl9" uri="http://www.bea.com/ns/weblogic/90"/>
+                </xmlfile>
+            </when>
+            <perform>
+                <iteration>
+                    <classification title="WebLogic EJB XML" severity="mandatory" effort="3">
+                        <link href="https://access.redhat.com/articles/1326823" description="Map weblogic-ejb-jar.xml Elements to the jboss-ejb3.xml Descriptor" />
+                    </classification>                        
+                    <!-- xslt description="JBoss EJB Descriptor (Windup-Generated)" template="transformations/xslt/weblogic9-ejb-to-jboss.xsl" extension="-jboss.xml"/-->
```

Comment:
I kept all xslt lines commented to have it for later if we want to update them (xslt for EAP6), but they could be removed. No strong opinion on this.